tests/test_unit_fixtures_settings.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+from devtools import debug
+
+from fractal_server.config import Settings
+
+DEFAULT_SETTINGS = Settings()
+
+
+def test_production_vs_test_defaults():
+
+    from fractal_server.config import get_settings
+
+    debug(DEFAULT_SETTINGS)
+
+    assert DEFAULT_SETTINGS.DB_ENGINE == "sqlite"
+    assert not DEFAULT_SETTINGS.SQLITE_PATH
+
+    settings = get_settings()
+    debug(settings)
+
+    if settings.DB_ENGINE == "sqlite":
```

Comment:
I like this test.
Can we make it more general? That is, it should always have an assertion, not only when using sqlite.

Reply:
The idea is: if it's not `sqlite`, than it must be `postgres`. So we are implicitly asserting that `DEFAULT_SETTINGS.DB_ENGINE == "postgres"`.
Making it explicit requires zero effort though.

Reply:
> The idea is: if it's not sqlite, than it must be postgres. So we are implicitly asserting that DEFAULT_SETTINGS.DB_ENGINE == "postgres".
> Making it explicit requires zero effort though.

OK, it makes sense. Having it explicit wouldn't hurt though, simply to avoid having this conversation again in a month.